data/remote/airspace/country/ES-ASP-National-SoaringWeb.txt.json

```diff
@@ -1,5 +1,5 @@
 {
   "description": "Spain Airspace from SoaringWeb.org",
-  "update": "2024-10-11",
-  "uri": "https://soaring.silentflight.ca/Airspace/ES/SUASpain202410.full.txt"
+  "update": "2025-02-13",
```

Comment:
The file says it's created 250211 and doesn't mention 250213, but I guess you know it's to be valid from 250113?